### PR TITLE
Composer should not to open when we select envelopes on draft

### LINKED ISSUE #6389: Multi-selecting drafts open modal editor
### Steps to reproduce

1. Go to Drafts folder
2. Click on three dots for selecting the first message
3. Click on checkbox to select further messages
4. See how modal editor window opens

### Expected behavior

Selecting a message with the checkbox selects the message

### Actual behavior

Selecting a message with the checkbox selects the message and opens it in the modal editor

### Mail app version

1.12.0

### Mailserver or service

_No response_

### Operating system

Docker image

### PHP engine version

PHP 8.0

### Web server

Apache (supported)

### Database

PostgreSQL

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by GretaD**
fixes #6389

### CODE REVIEW

**Comment by GretaD** [comment]
Please have a look and see if this makes sense

**Comment by ChristophWurst** [comment]
Does this work for you?

**Comment by GretaD** [comment]
> Does this work for you?

yes

File: src/components/Envelope.vue [hunk 1]
```diff
@@ -350,15 +350,19 @@ export default {
...
-			if (this.draft) {
-				this.$store.dispatch('showMessageComposer', {
-					data: {
-						...this.data,
-						draftId: this.data.databaseId,
-					},
-					templateMessageId: this.data.databaseId,
-				})
+		onClick(event) {
+			if (event) {
```
Comment (ChristophWurst) [inline src/components/Envelope.vue:354]: I don't understand why `event` would be truthy only in some cases. Don't we always get an event object from Vue?

Comment (GretaD) [reply src/components/Envelope.vue:354]: i honestly dont know, if i do `onClick(event) {
 event.preventDefault() }`
i doesnt work, if i add the if statement, it works, maybe my approach is wrong

Comment (ChristophWurst) [reply src/components/Envelope.vue:354]: Multiselect works now, but you can't open a draft anymore. 

The `event` is never falsy. So `onClick` always goes into the `if` branch, never `else`.

**Pull request merged**